supporter-product-data/cloudformation/dynamo-tables.yaml

```diff
@@ -15,66 +15,128 @@ Conditions:
   CreateDevOrUatResources: !Not [ !Equals [ !Ref "Stage", "PROD"]]
 
 Resources:
-  ProvisionedTable:
+  ProdTable:
     Condition: CreateProdResources
     Type: "AWS::DynamoDB::Table"
     Properties:
       AttributeDefinitions:
         - AttributeName: "identityId"
           AttributeType: "S"
-        - AttributeName: "ratePlanId"
+        - AttributeName: "subscriptionName"
           AttributeType: "S"
       KeySchema:
         # https://www.dynamodbguide.com/key-concepts/#primary-key
         # partition key
         - AttributeName: "identityId"
           KeyType: "HASH"
         # sort key
-        - AttributeName: "ratePlanId"
+        - AttributeName: "subscriptionName"
           KeyType: "RANGE"
       ProvisionedThroughput:
-        ReadCapacityUnits: 200
-        WriteCapacityUnits: 10
+        ReadCapacityUnits: 20
+        WriteCapacityUnits: 5
       SSESpecification:
         SSEEnabled: true
       TableName: !Sub SupporterProductData-${Stage}
       TimeToLiveSpecification:
-        AttributeName: "termEndDate"
+        AttributeName: "expiryDate"
         Enabled: true
-  OnDemandTable:
+
+  WriteCapacityScalableTarget:
```

Comment:
I have made the write capacity of the prod table autoscaling. This is because when we run a full sync it uses a massive amount of write capacity but in normal use when it is only running incremental updates 5 is easily enough.

Reply:
I don't think the autoscaling quite works or at least not quick enough, Mario G and I were investigating alarms in manage-frontend which turned out to be due to a full sync that was running.  It had scaled up once already but I had to scale again manually to stop the alarms.  Not sure if it's better to document it (somewhere?) as something to amend manually during a full sync.

Reply:
The autoscaling works fine now that I have updated the retry config of the Dynamo client to back off more aggressively and retry for longer. 
I've run multiple full syncs since then and they all completed successfully without errors.